public/app/features/explore/state/explorePane.ts

```diff
@@ -131,9 +152,16 @@ export function initializeExplore(
         queries,
         range,
         originPanelId,
+        datasourceInstance: instance,
+        history,
       })
     );
     dispatch(updateTime({ exploreId }));
+
+    if (instance) {
+      dispatch(runQueries(exploreId));
```

Comment:
Before the runQueries was hidden inside the `loadExploreDatasourcesAndSetDatasource` and triggered by `importQueries` switch which did not make much sense.